projects/exui-common-lib/src/lib/components/tc-confirm/tc-confirm.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="govuk-width-container">
+  <main role="main" class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <div class="column-two-thirds">
+          <h1 class="govuk-heading-xl">Using this service</h1>
+          <h2 class="govuk-heading-m">Terms and conditions</h2>
+          <p>
+            You have read and understood the
+            <a
+              [routerLink]="'/terms-and-conditions'"
```

Comment:
You could have this as `routerLink="/terms-and-conditions"`

Reply:
Well spotted, thanks!